data/scripts.js

```diff
@@ -114,7 +114,10 @@ let BattleScripts = {
 			for (const dancer of dancers) {
 				if (this.faintMessages()) break;
 				this.add('-activate', dancer, 'ability: Dancer');
-				this.runMove(move.id, dancer, 0, this.getAbility('dancer'), undefined, true);
+				// @ts-ignore
```

Comment:
`// @ts-ignore - target can't be null because <reason>`

ditto below. Documenting why youre ignoring will make it easier in the future to confidently change this to `target!.side` in the native Typescript etc. ditto below

Reply:
Done